src/sonic-config-engine/tests/test_frr.py

```diff
@@ -23,13 +23,22 @@ def tearDown(self):
 
     def run_script(self, argument, check_stderr=False):
 #        print '\n    Running sonic-cfggen ' + argument
+        write_output = False
+        if '-o' in argument:
+            write_output = True
+            output_file = argument[-1]
```

Comment:
How to guarantee '-o' is the last argument?

Reply:
I changed to different solution.